vyxal/transpile.py

```diff
@@ -493,7 +493,7 @@ def transpile_lambda(
             "this = self;",
             indent + 1,
         )
-        + indent_str("ctx.function_stack.append(this)", indent + 1)
+        + indent_str("ctx.function_stack.append(this);", indent + 1)
```

Comment:
Why a semicolon?

Reply:
Also remnants of experimentation and debugging